instrumentation/kafka/kafka-clients/kafka-clients-common-0.11/library/src/main/java/io/opentelemetry/instrumentation/kafkaclients/common/v0_11/internal/KafkaProducerAttributesExtractor.java

```diff
@@ -36,6 +38,11 @@ public void onStart(
     if (request.getRecord().value() == null) {
       attributes.put(MESSAGING_KAFKA_MESSAGE_TOMBSTONE, true);
     }
+
+    String bootstrapServers = request.getBootstrapServers();
+    if (bootstrapServers != null) {
+      attributes.put(MESSAGING_KAFKA_BOOTSTRAP_SERVERS, bootstrapServers);
+    }
```

Comment:
The latest core api expressly allows for `@Nullable` parameters (and it's internally guarded).
```suggestion
    attributes.put(MESSAGING_KAFKA_BOOTSTRAP_SERVERS, bootstrapServers);
```